### ISSUE TITLE
{Linked:Bug33178108}While navigating onto the 'End of chat history' text in the new agent chat, Narrator announces an additional word "note" (End of chat history, note)

### ISSUE THREAD

**Issue by Kjasnani7** [microsoft/BotFramework-WebChat#5854]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

When screen reader users navigate to the 'End of chat history' text in the new agent chat using Narrator, the announcement includes an unexpected and irrelevant word: “note”. This behavior introduces auditory noise and may cause confusion about whether there is additional context, annotation, or system feedback associated with the message.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://www.office.com/chat?auth=2

### How to reproduce the issue?

1. Launch the application using the URL: https://www.office.com/chat?auth=2 and login using v-credentials.
2. Turn on Narrator or NVDA
3. "Microsoft Copilot" page appears on the screen.
4. TAB to "Create Agent" in the right panel and press ENTER.
5. "Copilot Studio" dialog appears on the screen.
6. TAB to "Describe" tab and press ENTER.
7. Send Agent name which want to create.
8. TAB to Agent section which newly appeared at right side.
9. Turn on screen reader.
10. TAB to 'End of the Chat history".
11. Observe while navigating onto the 'End of chat history' text in the new agent chat, Narrator announces an additional word "note" (End of chat history, note).

### What do you expect?

While navigating onto the 'End of chat history' text in the new agent chat, Narrator should not announce additional word "note" (End of chat history, note)

### What actually happened?

While navigating onto the 'End of chat history' text in the new agent chat, Narrator announces an additional word "note" (End of chat history, note)
Note: issue is reproducing on win32 app as well.

### MAS reference

https://aka.ms/MAS1.3.1

### Adaptive Card JSON

```json

```

### Additional context

<img width="1292" height="608" alt="Image" src="https://github.com/user-attachments/assets/53f1a425-a19b-4edd-94a0-ed063556a516" />
https://github.com/user-attachments/assets/a908b875-5266-4242-8f73-18060cdeb15e
https://github.com/user-attachments/assets/3e00a937-bc9c-4e31-a5bc-e583cc57a765

https://github.com/user-attachments/assets/47a2bf17-5b36-4f05-a5a5-c7c4955cc350
